### ISSUE TITLE
What permissions are provided by the rbac.yaml file?

### ISSUE THREAD

**Issue by timubrns** [ParadigmAI/paradigm#1]
I find that that rbac.yaml is applied during installation. What is the purpose for this?

**Comment by mattchase17** [comment]
It defines the Kubernetes Service Account, ClusterRole, and a ClusterRoleBinding.

ServiceAccount named "paradigm-workflow" is created in the "paradigm" namespace. Service accounts provide an identity for processes that run in a Pod.
ClusterRole named "paradigm-workflow" is created with the following permissions:
    - For the "" (core) API group:
    - Permission to create, get, list, update, delete, and patch services in any namespace.
    - For the "apps" API group:
    - Permission to create, get, list, update, delete, and patch deployments in any namespace.
ClusterRoleBinding named "paradigm-workflow" is created which binds the "paradigm-workflow" ClusterRole to the "paradigm-workflow" ServiceAccount in the "paradigm" namespace.

In summary, any pod that is run with the "paradigm-workflow" ServiceAccount in the "paradigm" namespace will have permissions to create, get, list, update, delete, and patch services and deployments in any namespace within the Kubernetes cluster. It's important to note that these permissions are quite broad and could potentially be a security risk if the ServiceAccount is compromised or misused.

**Comment by timubrns** [comment]
Why is ClusterRoleBinding needed?
